layouts/index.html

```diff
@@ -92,17 +99,19 @@ <h2 class="title is-2 has-text-centered">About</h2>
 <!-- Let's chat, shall we? -->
 <div class="section" id="contact">
     <div class="container has-text-centered">
-    <h2 class="title is-2">Contact</h2>
+    <h2 class="title is-2">{{ i18n "CONTACT_TITLE" . }}</h2>
         <div class="markdown">
-            {{ range .Data.Pages }}
-                {{if eq .Title "Contact" }}
-                        {{.Content}}
-                    {{end}}
-                {{ end }}
+          {{ range .Data.Pages }}
```

Comment:
I would recommend having a look at `.Site.GetPage` etc. to improve this. Also see the headless bundles in Hugo if all you want is compose the home page with some content sections.